.github/workflows/python-package.yml

```diff
@@ -53,14 +53,15 @@ jobs:
         python -m pip install .[test]
     - name: Test source code and docs
       run:
-        cd tests;
```

Comment:
Do you have to delete `cd tests;`? With the default configuration, running `pytest` puts the working directory on the Python import path and so `import uncertainties` uses the files in the repo instead of the ones installed in site-packages. If there is an error in the packaging configuration such that a necessary file is not included in the package, the tests will still pass if they test with the files in the repo instead of the with the installed package. There are other ways to change pytest's behavior but just doing `cd tests` is a simple way to make sure the package is tested.

Reply:
I need to understand this more. I think I would prefer to configure the `pytest` runs more explicitly rather than relying on monkeying around with `cd` and the implicit "path".

Reply:
That is fine but we can do that in a separate PR? For a long time, the standard way to do this was to use a `src/` directory at the top level of the repo with the package inside of it. More recently pytest added some alternative ways of importing test modules instead of putting the working directory on the Python path. They come with some caveats though that we would have to assess. They are documented here:

https://docs.pytest.org/en/stable/explanation/pythonpath.html

Reply:
Yes, happy to discuss/do this in a separate PR. I'll pull that change out of this PR when I get a chance.